R/add_sda_target.R

```diff
@@ -46,6 +46,7 @@ add_sda_target <- function(data,
     "loan_size_outstanding",
     "loan_size_credit_limit",
     "name_ald",
+    "sector",
```

Comment:
This seems like it shouldn't be crucial to me. I'll need to explore this a little bit.